docs/source/notebooks/05_Advanced_RavenPy_configuration.ipynb

```diff
@@ -4,25 +4,20 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    print(config)
</pre><p>I'd do something like</p><pre class="ql-syntax" spellcheck="false">print([f.name for f in config])
</pre><p>and skip the next cell. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Good call, changed.</p>